src/fusiondls/postprocessing.py

```diff
@@ -11,19 +11,26 @@ class FrontLocationScan:
     useful statistics: detachment window and unstable region size.
     """
 
-    def __init__(self, store):
-        num_locations = len(store["Sprofiles"])
+    def __init__(self, SimulationOutputs):
+        out = SimulationOutputs
```

Comment:
I would reserve `CamelCase` for class/type names and use `snake_case` for variable names, with exceptions for things like physical quantities, where `t=time` and `T=temperature` may appear in the same equation (although in that case I'd lean towards just writing out `time` and `temperature` in full to remove all ambiguity). As this is very similarly named to `SimulationOutput`, it could be easily confused here for the class instead of the local variable. ([PEP8 recommendation](https://peps.python.org/pep-0008/#function-and-variable-names))

```suggestion
    def __init__(self, out: SimulationOutput):
```

Reply:
Thanks, this is useful feedback. Fixed